src/hooks/messages/useSafeMessageNotifications.ts

```diff
@@ -24,6 +25,10 @@ const SafeMessageNotifications: Partial<Record<SafeMsgEvent, string>> = {
   [SafeMsgEvent.SIGNATURE_PREPARED]: 'The message was successfully confirmed.',
 }
 
+const isUserRejection = (error: Error) => {
```

Comment:
Maybe we can move this function into one of the utils files like `ethers-utils.ts` or create a new one for error utils so we don't have to define it twice and can reuse the logic in other places.